client/src/components/screens/Patient/components/FilesTab/index.tsx

```diff
@@ -4,41 +4,37 @@ import {
 } from 'antd';
 import { DownOutlined } from '@ant-design/icons';
 import intl from 'react-intl-universal';
+import { filesColumnsIntl } from 'store/graphql/files/models';
+import { getFilesData } from 'store/graphql/files/actions';
+import { useSelector } from 'react-redux';
+import { formatBytes } from 'utils';
 import Api from '../../../../../helpers/api';
-import { TaskResponse } from '../../../../../helpers/search/types';
+import { PatientResponse } from '../../../../../helpers/search/types';
 import MetadataModal from './metadataModal';
+import { State } from '../../../../../reducers';
+import { ParsedPatientData } from '../../../../../helpers/providers/types';
+import EmptyCard from './components/EmptyCard';
 
 import './styles.scss';
 
-const fileInfo: {[key: string]: any} = require('./info.json');
-
 const getURL = async (url: string) => {
   const data: any = await Api.getFileURL(url);
   return data.payload.data.url;
 };
 
-const FilesTab: React.FC = () => {
-  const { Patient } = fileInfo;
+type FilesTabProps = {
+  userToken: string;
+};
 
+const FilesTab: React.FC<FilesTabProps> = () => {
   const dataSource: any[] = [];
+  const patient = useSelector((state: State) => state.patient.patient.parsed) as ParsedPatientData;
   const [isOpen, setIsOpenModal] = useState<boolean>(false);
   const [documentReference, setDocumentReference] = useState<string>('');
-
-  const getFileSize = (size: number) => {
-    let newSize = size;
-    if (size >= 1000 && size < 10 ** 6) {
-      newSize = size / 1000;
-      return `${newSize} ${intl.get('screen.patient.details.file.size.kb')}`;
-    } if (size >= 10 ** 6 && size < 10 ** 9) {
-      newSize = size / 10 ** 6;
-      return `${newSize} ${intl.get('screen.patient.details.file.size.mb')}`;
-    } if (size >= 10 ** 9 && size < 10 ** 12) {
-      newSize = size / 10 ** 9;
-      return `${newSize} ${intl.get('screen.patient.details.file.size.gb')}`;
-    }
-
-    return `${newSize} ${intl.get('screen.patient.details.file.size.b')}`;
-  };
+  const { loading: loadingData, results: data } = getFilesData({ patientId: patient.id })();
+  if (loadingData) {
```

Comment:
In the Apollo query, you return the previous data is the current data is not available yet. Here you check if its loading and if so return null (regardless of anything else).

Maybe check what is the desired behavior of the portal in this situation. 
- If you have some data (previous) and is loading, do you still what to show the component with it until the new data comes in?

- or maybe, if loading and you have previous data, you just want to show an empty card, until the data comes in? (in that case maybe add a spinner?)

- or maybe this is the desired behavior (return null if loading) and then disregard this comment :)